IO/NativeSignalHandlers/UnixHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Mono.Unix;
+using Mono.Unix.Native;
+using System.Threading.Tasks;
+
+namespace LocalAdmin.V2.IO.NativeSignalHandlers
+{
+    /// <summary>
+    ///     Native signal processing on Unix systems.
+    /// </summary>
+    internal sealed class UnixHandler : INativeSignalHandler
+    {
+        public static readonly UnixHandler Handler;
+        private static readonly UnixSignal[] signals;
+
+        static UnixHandler()
```

Comment:
You can initialize those above, no need for manually making a static constructor

Reply:
Does this make sense?
From ECMA-334:
> `17.4.5.1` **Static field initialization**
The static field variable initializers of a class declaration correspond to a sequence of assignments that are
executed in the textual order in which they appear in the class declaration. If a static constructor (§17.11)
exists in the class, execution of the static field initializers occurs immediately prior to executing that static
constructor. Otherwise, the static field initializers are executed at an implementation-dependent time prior to
the first use of a static field of that class.

In this case, the initialization of the fields occurs before accessing the static constructor, and I don’t initialize them, a static constructor initializes them instead. In another case, its initialization occurs before the first use of one of the static fields of this class, does it make sense to change something? This is more on the topic of how convenient it is, in which case it's convenient for me to use a static constructor. ¯\\_(ツ)_/¯

Reply:
Initializing in the declaration gives cleaner code and makes viewing things like array elements easier after doing for example "Go to declaration" on the variable in code

Reply:
Perhaps when you have a lot of code, especially with comment blocks, then it’s better for you, but not to me, the length of the variable name and gray blocks of code make my eyes watery and I prefer to use the static constructor, because I see all the values linearly at once, here it’s not, but I still prefer to use it, also these variables with a private access modifier, I don’t think that you’ll get lost in 30 lines of code.

Reply:
Resolved in https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/afc8b33f4dac4ece0e9b22d17701c63beea1854e